testbed/tests/assertions.py

```diff
@@ -12,5 +12,9 @@ def assert_set_get(obj, name, value):
 
 def assert_color(actual, expected):
     for component in ["r", "g", "b"]:
-        assert getattr(actual, component) == getattr(expected, component)
-    assert actual.a == approx(expected.a, abs=(1 / 255))
+        assert getattr(actual, component) == getattr(
+            expected, component
+        ), f"actual={actual} != expected={expected}"
+    assert actual.a == approx(
+        expected.a, abs=(1 / 255)
+    ), f"actual={actual} != expected={expected}"
```

Comment:
pytest is supposed to print the actual and expected values automatically, but I guess it didn't work because the ["assertion rewriting"](https://docs.pytest.org/en/stable/how-to/writing_plugins.html#assertion-rewriting) feature is only on by default for code directly inside of test modules. 

I previously tried fixing this by adding the following to `tests/__init__.py`, but it didn't have any effect, at least on Windows:
  ```
  import pytest
  pytest.register_assert_rewrite("tests")
  ```
I think we should get to the bottom of this and fix it properly rather than cluttering all of our assertions like this.